mindmeld/components/dialogue.py

```diff
@@ -820,6 +821,19 @@ def _validate(self, request, slot):
 
             else:
                 # gazetteer validation
+
+                # payload format for entity feature extractors:
+                # tuple(query (Query Object), list of entities, entity index)
+                if request.entities:
```

Comment:
is this logic similar to line 808?

Reply:
No the follow-up logic is a bit different.